<ui version="4.0" >
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Plan Treatment</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" colspan="5" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Add these items to the estimate?</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="5" >
    <widget class="QScrollArea" name="scrollArea" >
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>161</height>
      </size>
     </property>
     <property name="widgetResizable" >
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>573</width>
        <height>390</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout" >
       <item>
        <widget class="QFrame" name="frame" >
         <property name="frameShape" >
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow" >
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="label_4" >
     <property name="text" >
      <string>Total Fee</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QDoubleSpinBox" name="fee_doubleSpinBox" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="prefix" >
      <string>Â£</string>
     </property>
     <property name="maximum" >
      <double>3000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="2" >
    <spacer name="horizontalSpacer" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>160</width>
       <height>26</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="3" >
    <widget class="QPushButton" name="later_pushButton" >
     <property name="text" >
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4" >
    <widget class="QPushButton" name="yes_pushButton" >
     <property name="text" >
      <string>Yes, add these Items Now</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>yes_pushButton</tabstop>
  <tabstop>later_pushButton</tabstop>
  <tabstop>scrollArea</tabstop>
  <tabstop>fee_doubleSpinBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>yes_pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>520</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel" >
     <x>442</x>
     <y>261</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>later_pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>254</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel" >
     <x>364</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
